src/index.js

```diff
@@ -6,7 +6,7 @@ import { BrowserRouter } from "react-router-dom";
 
 ReactDOM.render(
   <React.StrictMode>
-    <BrowserRouter basename="brewed-store">
+    <BrowserRouter>
```

Comment:
nothing wrong here but there is a common convention that people follow is that they write Router instated of browser router so when you are importing give browserrouter an alias of router.